src/Window.cpp

```diff
@@ -1090,7 +1090,8 @@ void Window::dispatchCommand(QString command) {
 QString Window::executeCommand(QString command) {
     QStringList tokens = command.split(" ", QString::SkipEmptyParts);
     if (tokens.size() != 1) {
-        return INVALID;
+        if (! (tokens.size() == 2 && tokens.at(0) == "moveForward" ))
+            return INVALID;
     }
```

Comment:
I think it's slightly cleaner to check the easy case and the harder case separately, so that you can use the `function` variable in the latter, rather than `tokens.at(0)`. Also, my preference is to *always* use braces, just to be safe.
```cpp
if (tokens.size() < 1 || tokens.size() > 2) {
  return INVALID;
}
QString function = tokens.at(0);
if (tokens.size() == 2 && function != "moveForward") {
    return INVALID;
}
```